test/Serialization/comments-hidden.swift

```diff
@@ -0,0 +1,59 @@
+// Test the case when we compile normally
+//
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: %target-swift-frontend -module-name comments -emit-module -emit-module-path %t/comments.swiftmodule -emit-module-doc -emit-module-doc-path %t/comments.swiftdoc %s
+// RUN: %target-swift-ide-test -print-module-comments -module-to-print=comments -source-filename %s -I %t | FileCheck %s -check-prefix=NORMAL
+
+// Test the case when we compile with -enable-testing
+//
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: %target-swift-frontend -enable-testing -module-name comments -emit-module -emit-module-path %t/comments.swiftmodule -emit-module-doc -emit-module-doc-path %t/comments.swiftdoc %s
+// RUN: %target-swift-ide-test -print-module-comments -module-to-print=comments -source-filename %s -I %t | FileCheck %s -check-prefix=TESTING
+
+/// PublicClass Documentation
+public class PublicClass {
+  /// Public Function Documentation
+  public func f_public() { }
+  /// Internal Function Documentation
+  internal func f_internal() { }
+  /// Private Function Documentation
+  private func f_private() { }
+}
+
+/// InternalClass Documentation
+internal class InternalClass {
+  /// Internal Function Documentation
+  internal func f_internal() { }
+  /// Private Function Documentation
+  private func f_private() { }
+}
+
+/// PrivateClass Documentation
+private class PrivateClass {
+  /// Private Function Documentation
+  private func f_private() { }
+}
+
+// NORMAL-NOT: PrivateClass Documentation
```

Comment:
Do you know if `NORMAL-NOT: Private` matches `PrivateMoreTokensHereWithoutWhitespace`?

I assume based on your comment that it does, but I was uncertain.